src/styles/components/_task-card.scss

```
@@ -0,0 +1,9 @@
+@use '../abstracts/variables' as *;
```

Can we use relative import here ?
```suggestion
@use 'styles/abstracts/variables' as *;
```